<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="5" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-08-24T15:18:07Z" name="OrageHRM" finished-at="2020-08-24T15:19:29Z" duration-ms="81786">
    <groups>
    </groups>
    <test started-at="2020-08-24T15:18:07Z" name="OragneHRM Test" finished-at="2020-08-24T15:19:29Z" duration-ms="81786">
      <class name="com.orangehrm.Database.LoginfromDatabase">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.orangehrm.Database.LoginfromDatabase@11392934]" started-at="2020-08-24T15:19:19Z" name="setup" finished-at="2020-08-24T15:19:24Z" duration-ms="4847" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="setup()[pri:0, instance:com.orangehrm.Database.LoginfromDatabase@11392934]" started-at="2020-08-24T15:19:24Z" name="setup" finished-at="2020-08-24T15:19:28Z" duration-ms="4218" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.orangehrm.Database.LoginfromDatabase@11392934]" started-at="2020-08-24T15:19:28Z" name="close" finished-at="2020-08-24T15:19:29Z" duration-ms="782" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.orangehrm.Database.LoginfromDatabase -->
      <class name="com.orangehrm.TestCases.LoginDDTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:18Z" name="setup" finished-at="2020-08-24T15:18:24Z" duration-ms="5540" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="logindatadriven(java.lang.String, java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:25Z" name="logindatadriven" data-provider="LoginData" finished-at="2020-08-24T15:18:39Z" duration-ms="14036" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[automation]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Automation@123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logindatadriven -->
        <test-method signature="logindatadriven(java.lang.String, java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:39Z" name="logindatadriven" data-provider="LoginData" finished-at="2020-08-24T15:18:43Z" duration-ms="3848" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Xyztqe]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[xwffw]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.orangehrm.TestCases.LoginDDTest.logindatadriven(LoginDDTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logindatadriven -->
        <test-method signature="logindatadriven(java.lang.String, java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:43Z" name="logindatadriven" data-provider="LoginData" finished-at="2020-08-24T15:18:47Z" duration-ms="3652" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[fsdfsf]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3654dfdf]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.orangehrm.TestCases.LoginDDTest.logindatadriven(LoginDDTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logindatadriven -->
        <test-method signature="logindatadriven(java.lang.String, java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:47Z" name="logindatadriven" data-provider="LoginData" finished-at="2020-08-24T15:18:55Z" duration-ms="8329" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[automation]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Automation@123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logindatadriven -->
        <test-method signature="logindatadriven(java.lang.String, java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:55Z" name="logindatadriven" data-provider="LoginData" finished-at="2020-08-24T15:18:59Z" duration-ms="3619" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[dffddf]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[32323ffasf]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.orangehrm.TestCases.LoginDDTest.logindatadriven(LoginDDTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logindatadriven -->
        <test-method signature="logindatadriven(java.lang.String, java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:18:59Z" name="logindatadriven" data-provider="LoginData" finished-at="2020-08-24T15:19:02Z" duration-ms="3565" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Peter]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123DFf]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.orangehrm.TestCases.LoginDDTest.logindatadriven(LoginDDTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logindatadriven -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.orangehrm.TestCases.LoginDDTest@6d4d66d2]" started-at="2020-08-24T15:19:02Z" name="close" finished-at="2020-08-24T15:19:03Z" duration-ms="702" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.orangehrm.TestCases.LoginDDTest -->
      <class name="com.orangehrm.TestCases.PersonalDetailsTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.orangehrm.TestCases.PersonalDetailsTest@2a265ea9]" started-at="2020-08-24T15:19:03Z" name="setup" finished-at="2020-08-24T15:19:07Z" duration-ms="4549" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="editpersonaldetails()[pri:0, instance:com.orangehrm.TestCases.PersonalDetailsTest@2a265ea9]" started-at="2020-08-24T15:19:07Z" name="editpersonaldetails" finished-at="2020-08-24T15:19:19Z" duration-ms="11194" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editpersonaldetails -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.orangehrm.TestCases.PersonalDetailsTest@2a265ea9]" started-at="2020-08-24T15:19:19Z" name="close" finished-at="2020-08-24T15:19:19Z" duration-ms="744" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.orangehrm.TestCases.PersonalDetailsTest -->
      <class name="com.orangehrm.TestCases.LoginTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.orangehrm.TestCases.LoginTest@54c5a2ff]" started-at="2020-08-24T15:18:08Z" name="setup" finished-at="2020-08-24T15:18:14Z" duration-ms="6234" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest()[pri:0, instance:com.orangehrm.TestCases.LoginTest@54c5a2ff]" started-at="2020-08-24T15:18:14Z" name="loginTest" finished-at="2020-08-24T15:18:18Z" duration-ms="3910" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.orangehrm.TestCases.LoginTest@54c5a2ff]" started-at="2020-08-24T15:18:18Z" name="close" finished-at="2020-08-24T15:18:18Z" duration-ms="661" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.orangehrm.TestCases.LoginTest -->
    </test> <!-- OragneHRM Test -->
  </suite> <!-- OrageHRM -->
</testng-results>
